src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: inline;
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
use scss loop for creating selectors for stars, make it possible to change active stars with changins stars--n selector.